docs/endpoints/commands.md

```diff
@@ -0,0 +1,11 @@
+#### Probe commands
+
+- Build - builds the specified file, module or project
+- Close Project - closes the specified project
+- Invoke Action Async - invokes the specified actions without waiting for it to finish
+- Invoke Action - invokes the specified actions and waits for it to finish
+- Open Project - opens the specified file as a project
+- Run - runs the specified configuration
```

Comment:
this is for running applications (main classes), description sounds like arbitrary run conf